src/local.js

```diff
@@ -9,7 +9,8 @@ export default (Config) => (Component) => {
     const defaultMapStateToProps = (state) => state;
     const ConnectComp = connect(
         Config.mapStateToProps || defaultMapStateToProps,
-        Config.mapDispatchToProps)( (props) => {
+        Config.mapDispatchToProps,
+        Config.mergeProps)( (props) => {
```

Comment:
Please also add an example in the Readme with the new config so people are aware of it and start using it. We should probably better organize the readme somehow as the content starts to grow.